src/commands.ts

```diff
@@ -235,6 +236,24 @@ export class CosmosDBCommands {
         }
     }
 
+    public static async createDocDBDocument(coll: DocDBCollectionNode, explorer: CosmosDBExplorer) {
+        const masterKey = coll.db.getPrimaryMasterKey();
+        const endpoint = coll.db.getEndpoint();
+        const client = new DocumentClient(endpoint, { masterKey: masterKey });
+        await new Promise((resolve, reject) => {
+            client.createDocument(coll.getCollLink(), { 'id': 'temporary' }, (err, result) => {
+                if (err) {
+                    reject(err.body);
+                }
+                else {
+                    resolve(result);
+                }
+            });
+        });
+        explorer.refresh(coll);
```

Comment:
I only do a refresh of the collection node and not open the temporary document. 
The reason was that if the user has any un-persisted changes on the document they are working on, we wouldn't want to overwrite that.